src/lib/assets/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ngPackage": {
+    "lib": {
+      "entryFile": "index.ts",
+      "umdModuleIds": {
+        "@covalent/http": "covalent.http",
+        "@ngx-translate/core": "ngx-translate.core"
```

Comment:
Looks like ngx translate is not be used here. Can we remove this?

Reply:
Done.